fastanvil/src/region.rs

```diff
@@ -25,6 +26,7 @@ pub(crate) const CHUNK_HEADER_SIZE: usize = 5;
 /// File). This does not concern itself with manipulating chunk data, users are
 /// expected to use `fastnbt` or other deserialization method to manipulate the
 /// chunk data itself.
+#[derive(Clone)]
```

Comment:
I was suprised this worked given `S` isn't marked as `Clone`, but turns out the derived `Clone` adds a bound for `S`, so `Region` won't implement it for eg `S = File` but will for `S = Cursor<Vec<u8>>`. I'm happy with this.